spring-boot-modules/spring-boot-properties-3/src/main/resources/application.properties

```diff
@@ -31,4 +31,13 @@ bael.property=prodValue
 environment.name=${OS}
 java.home.and.environment=${JAVA_HOME}+${OS}
 not.existing.system.property=${thispropertydoesnotexist}
-baeldung.presentation=${HELLO_BAELDUNG}. Java is installed in the folder: ${JAVA_HOME}
\ No newline at end of file
+baeldung.presentation=${HELLO_BAELDUNG}. Java is installed in the folder: ${JAVA_HOME}
+#---
+#camelCasing
+#profile.applicationProperties.name="Baeldung"
+#PascalCasing
+#profile.ApplicationProperties.name="Baeldung"
+#SnakingCasing
+#profile.application_properties.name="Baeldung"
+#KebabCasing
+profile.application-properties.name="Baeldung"
```

Comment:
profile.**application-**properties.name differs from the one mentioned in the draft:
https://drafts.baeldung.com/?p=159983&preview=true#basic-configuration

Reply:
Resolved this.